xml/System.Collections.Immutable/ImmutableList`1+Builder.xml

```diff
@@ -131,11 +131,14 @@
     </Member>
     <Member MemberName="BinarySearch">
       <MemberSignature Language="C#" Value="public int BinarySearch (T item);" />
-      <MemberSignature Language="ILAsm" Value=".method public hidebysig instance int32 BinarySearch(!T item) cil managed" />
+      <MemberSignature Language="ILAsm" Value=".method public hidebysig newslot virtual instance int32 BinarySearch(!T item) cil managed" FrameworkAlternate="dotnet-uwp-10.0;netcore-1.0;netcore-1.1" />
```

Comment:
if this was virtual for some versions, don't we need distinguished signatures for C# and VB?

Reply:
Ahh, ok ... so here's what's happening here; per this line in the c# formatter:

```csharp
if (modifiers == " virtual sealed") modifiers = "";
```

When a method is `virtual sealed`, it simply omits that modifier from the signature because it's not possible to override it (due to `public sealed class ImmutableList&lt;T&gt;.Builder`). So the only reason this difference in the signature formatters is becoming apparent now, is because of the new feature that separates signatures only when they differ.